packages/php/email-editor/tests/integration/Integrations/Core/Renderer/Blocks/Embed_Test.php

```
@@ -534,6 +534,172 @@ public function test_youtube_embed_falls_back_to_link_when_thumbnail_fails(): vo
+
+	/**
+	 * Test that a /embed/videoseries playlist URL does not build a broken thumbnail
+	 *
+	 * The "videoseries" token would otherwise be captured as a video ID, producing a
+	 * broken img.youtube.com/vi/videoseries thumbnail. It is excluded so the URL falls
+	 * through to the oEmbed lookup; WP core has no provider pattern for this form, so
+	 * it degrades to a graceful link fallback.
+	 */
+	public function test_youtube_playlist_embed_url_does_not_build_broken_thumbnail(): void {
+		// Block the outbound oEmbed discovery request so the test stays offline-deterministic.
+		$filter_callback = function ( $preempt, $args, $url ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)